### PR TITLE
Issue #15214: Migrated section 4.8.3.2 No C Style Array Declarations to whole config testing

### PR DESCRIPTION
**Pull request by Zopsss**
#15214

### CODE REVIEW

File: src/it/resources/com/google/checkstyle/test/chapter4formatting/rule4832nocstylearray/InputNoCstyleArrays.java [hunk 1]
```diff
@@ -22,38 +21,34 @@ public static void mainC(String aCStyle[]) // violation 'Array brackets at illeg
...
 
-    public Test getOldTests()[][]
-    // 2 violations above:
+    // 2 violations 3 lines below:
     //                    'Array brackets at illegal position.'
     //                    'Array brackets at illegal position.'
-    {
+    public Test getOldTests()[][] {
       return null;
     }
```
Comment (Zopsss) [inline src/it/resources/com/google/checkstyle/test/chapter4formatting/rule4832nocstylearray/InputNoCstyleArrays.java:45]: moved the comment outside of method so we don't cross the limit of `minLineCount`:

https://github.com/checkstyle/checkstyle/blob/2057ac975e1c9ad48fdad077afc0f75ae6bb23bc/src/main/resources/google_checks.xml#L338-L343

Comment (Zopsss) [inline src/it/resources/com/google/checkstyle/test/chapter4formatting/rule4832nocstylearray/InputNoCstyleArrays.java:45]: moved the comment outside of method so we don't cross the limit of `minLineCount`:

https://github.com/checkstyle/checkstyle/blob/2057ac975e1c9ad48fdad077afc0f75ae6bb23bc/src/main/resources/google_checks.xml#L338-L343

**Review by romani — APPROVED** [review]
ok to merge.

there are a bit refactoring on names of variables (lowercased) to avoid unrelated violations , I am ok with this.

CI are restarted.

**Review by rdiachenko — APPROVED** [review]
lgtm

**Comment by Zopsss** [comment]
semaphore can be ignored. @rdiachenko or @romani pls merge the pr

**Pull request merged**